lib/main_publiccloud.pm

```diff
@@ -60,20 +60,21 @@ sub load_maintenance_publiccloud_tests {
 }
 
 sub load_publiccloud_consoletests {
-    loadtest 'console/cleanup_qam_testrepos' unless get_var('PUBLIC_CLOUD_QAM');
-    loadtest 'console/openvswitch';
-    loadtest 'console/rpm';
-    loadtest 'console/openssl_alpn';
-    loadtest 'console/check_default_network_manager';
-    loadtest 'console/sysctl';
-    loadtest 'console/sysstat';
-    loadtest 'console/gpg';
-    loadtest 'console/sudo';
-    loadtest 'console/supportutils';
-    loadtest 'console/journalctl';
-    loadtest 'console/procps';
-    loadtest 'console/suse_module_tools';
-    loadtest 'console/libgcrypt' unless is_sle '=12-SP4';
+    my ($run_args) = @_;
+    loadtest 'console/cleanup_qam_testrepos', run_args => $run_args unless get_var('PUBLIC_CLOUD_QAM');
+    loadtest 'console/openvswitch', run_args => $run_args;
+    loadtest 'console/rpm', run_args => $run_args;
```

Comment:
I just randomly checked `console/rpm` it does not accept args . https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/console/rpm.pm#L31 . I am against passing it just to be generic ...

Reply:
ah so you need it in `opensusebasetest->post_fail_hook` ?

Reply:
> ah so you need it in `opensusebasetest->post_fail_hook` ?

Yes. See [poo#95780](https://progress.opensuse.org/issues/95780).

Reply:
ok